tests/hipcc_check.cpp

```diff
@@ -10,8 +10,26 @@ int main() {
   constexpr int blocksize = 64;
   constexpr int size = gridsize * blocksize;
   int *d_buf;
-  hipMalloc(&d_buf, size * sizeof(int));
+  hipHostMalloc(&d_buf, size * sizeof(int));
   hipLaunchKernelGGL(squares, gridsize, blocksize, 0, 0, d_buf);
   hipDeviceSynchronize();
+
+  // Check results.
+  int mismatches_count = 0;
+  bool failed = false;
```

Comment:
`failed` not used anywhere?

Reply:
Whoops, leftover from checking that single value. I switched to `mismatches_count`.